### DISCUSSION TITLE
Muffin - Progress Report Discussion

### DISCUSSION THREAD

**Discussion by ramy-att** [SOEN345-WINTER2024/discussion#15] [General]
Muffin - Progress Report Discussion: Add bugs here

**Comment by ramy-att** [comment]
# RedReader Bug:

- Device Info: Used Pixel 4A, android version 2024010900
- Version tested: V1.23.1 stable
- Stacktrace: Bug does not cause a crash
- Duplicate of https://github.com/QuantumBadger/RedReader/issues/1143

## How to reproduce:
1. Launch the app
2. Open any subreddit
3. Open any picture
4. Swipe screen left to show the sidebar
5. The screen goes black and shows scrollbars

## Actual behavior:

![thumbnail_Screenshot_20240321-213752](https://github.com/SOEN345-WINTER2024/discussion/assets/83047251/5cce2120-7f7d-49c8-90fd-1d4e0979b089)

## Expected behavior:

- Sidebar shows up with all the options

Duplicate of https://github.com/QuantumBadger/RedReader/issues/1143

**Reply by TarekElalfi** [reply]
Very nice finding this bug. The app also crashes which is great.

**Comment by YoussefMorcos** [comment]
Good job!

**Comment by TarekElalfi** [comment]
###  SmartTube Bug:

- Stacktrace: The Bug does not make the app crash
- Device Information: Used Android 8.1 (Andriod TV)
- Version tested: Version 21.54 Stable
- Patch: Could not issue a pull request
- Duplicate: This bug is new and not duplicated

**How to reproduce:**
1. Launch the app
2. select a Video from any row other than the topmost 

**Actual behaviour:**

![1](https://github.com/SOEN345-WINTER2024/discussion/assets/98570182/ac6d966a-e0c9-4aed-a522-0ac3a8186342)


**Expected behaviour:**
The dashboard (Search and profile Icons should Appear) as shown below:

![2](https://github.com/SOEN345-WINTER2024/discussion/assets/98570182/d04b0b88-32db-4407-9fec-6db570ea125a)

**Reply by ramy-att** [reply]
Nice find!!
LGTM
